jdbc/JDBCReleaseTest.java

```diff
@@ -272,16 +252,13 @@ static void simplePreparedTest2() throws SQLException
 		Connection db = DriverManager.getConnection(url, psql_username, "");
 		Statement stmt = db.createStatement();
 		stmt.executeUpdate("SET citus.enable_repartition_joins TO true;");
-		stmt.executeUpdate("SET citus.large_table_shard_count TO 2;");
 		stmt.executeUpdate("SET citus.task_executor_type TO '" + task_executor_type + "'" );
 
-		PreparedStatement st = db.prepareStatement("insert into orders values ('')");
-		PreparedStatement st2 = db.prepareStatement("SELECT sum(o_totalprice) from orders where o_orderkey::text like '%88'");
+		PreparedStatement st = db.prepareStatement("SELECT sum(o_totalprice) from orders where o_orderkey::text like '%88'");
 
 		for (int i = 0; i < 3; ++i)
 		{
-			System.out.println(st.executeUpdate());
```

Comment:
couldn't follow where did `st` go

Reply:
The original
```java
PreparedStatement st = db.prepareStatement("insert into orders values ('')");
```
Seems like a wrong query to me so I removed it, and renamed st2 as st.
Am I missing something here?